hello-world/README.md

```diff
@@ -329,7 +330,7 @@ while True:
     time.sleep(1)
 ```
 
-Now open a **new** command line terminal and go to the `hello-world` directory.
+Now open a **new** command line terminal and go to the `./hello-world/python` directory.
```

Comment:
Can you make this sentence and the node one "Open a terminal window and navigate" consistent. One says Command Line terminal and the other terminal window.

Reply:
"command line terminal" was a preexisting issue, but I was happy to fix while in there.  That's all done now.  We're going with "new terminal" everywhere.